### PR TITLE
Proposal to have a fr gender inclusive a bit more light

### PR DESCRIPTION
**Pull request by aHenryJard**
French improvement proposal:
- Fixing sentence that are not gender neutral yet
- proposing some more light ways to be gender neutral by using a bit more "personne qui"

### CODE REVIEW

File: content/version/2/0/code_of_conduct.fr.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ ouverte, accueillante, diversifiée, inclusive et saine.
 Exemples de comportements qui contribuent à créer un environnement positif :
 
 * Faire preuve d'empathie et de bienveillance envers les autres
-* Être respectueux des opinions, points de vue et expériences divergents
+* Respecter les opinions, points de vue et expériences divergentes
```
Comment (aHenryJard) [inline content/version/2/0/code_of_conduct.fr.md:27]: This sentence was not gender inclusive

File: content/version/2/0/code_of_conduct.fr.md [hunk 2]
```diff
@@ -67,13 +67,13 @@ inacceptables peuvent être signalés aux dirigeant·e·s de la communauté resp
...
 Toutes les plaintes seront examinées et feront l'objet d'une enquête rapide et équitable.
 
-Tou·te·s les dirigeant·e·s de la communauté sont tenu·e·s de respecter la vie privée et la sécurité des
+Toutes les personnes dirigeantes de la communauté sont tenues de respecter la vie privée et la sécurité des
 personnes ayant signalé un incident.
 
 ## Directives d'application
 
-Les dirigeant·e·s de communauté suivront ces directives d'application sur l'impact communautaire afin de déterminer
-les conséquences de toute action qu'ils jugent contraire au présent code de conduite :
```
Comment (aHenryJard) [inline content/version/2/0/code_of_conduct.fr.md:76]: this was not gender inclusive

**Review by oleg-nenashev — APPROVED** [review]
My French is terrible but the new version looks better

**Comment by aHenryJard** [comment]
Way too old, farewell :wave:

**Pull request closed without merging**